docs/http/http-spans.md

```diff
@@ -66,27 +66,27 @@ and various HTTP versions like 1.1, 2 and SPDY.
 
 ## Name
 
+**Status**: [Mixed][DocumentStatus]
+
```

Comment:
I think I prefer "Stable, Unless otherwise specified", since Mixed makes me want to see each subsection marked as either Stable or Experimental.

(and page is now already "Stable, Unless otherwise specified")

```suggestion
```